pkg/v3/plugin/ocr3.go

```diff
@@ -0,0 +1,87 @@
+package plugin
+
+import (
+	"context"
+	"errors"
+
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+
+	ocr2keepersv3 "github.com/smartcontractkit/ocr2keepers/pkg/v3"
+)
+
+type InstructionStore interface{}
+
+type SamplingStore interface{}
```

Comment:
Can these two interfaces be unexported?

Reply:
I'll do that.

Reply:
Actually, since they are defined in the build hooks, they are needed for now, right?

Reply:
I think they'll need to be moved to the same place as the result store interface eventually.